spec/index.adoc

```diff
@@ -922,8 +922,8 @@ In order for a WebAssembly module to be usable as the code for the canister, it
 * If it exports a function called `canister_inspect_message`, the function must have type `+() -> ()+`.
 * If it exports a function called `canister_heartbeat`, the function must have type `+() -> ()+`.
 * If it exports a function called `canister_global_timer`, the function must have type `+() -> ()+`.
-* If it exports any functions called `canister_update <name>` or `canister_query <name>` for some `name`, the functions must have type `+() -> ()+`.
-* It may not export both `canister_update <name>` and `canister_query <name>` with the same `name`.
+* If it exports any functions called `canister_update <name>`, `canister_query <name>`, or `canister_composite_query <name>` for some `name`, the functions must have type `+() -> ()+`.
```

Comment:
Did we want to mark the new functionality as experimental somewhere?

Reply:
Added a note at two places.